public/.well-known/manifest.webmanifest

```
@@ -2,7 +2,7 @@
   "id": "/",
   "short_name": "IWA KS",
   "name": "Kitchen Sink IWA",
-  "version": "0.0.0",
+  "version": "0.0.23",
```

```suggestion
  "version": "0.0.0",
```

Reverting as the version is built automatically and shouldn't be updated here.